libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -94,6 +94,14 @@ void AP_CRSF_Telem::setup_custom_telemetry()
         return;
     }
 
+    // check if passthru already assigned
+    const int8_t frsky_port = AP::serialmanager().find_portnum(AP_SerialManager::SerialProtocol_FrSky_SPort_Passthrough,0); 
+    if (frsky_port != -1) {
+        gcs().send_text(MAV_SEVERITY_CRITICAL, "CRSF: passthrough telemetry conflict on SERIAL%d",frsky_port);
+       _custom_telem.init_done = true;
```

Comment:
just overthinking wheter _custom_telem.init_done = true is right or not, I guess it's fine now but not "proper", setting it true does not guarantee full initialization of the custom telemetry subsystem so if in the future we add init actions that are required from other methods we might have unexpected results.

Reply:
@andyp1per any thoughts on this?

Reply:
Without this it will infinitely loop